wemake_python_styleguide/violations/oop.py

```diff
@@ -265,12 +265,17 @@ class MethodWithoutArgumentsViolation(ASTViolation):
         class Test:
             def method(self): ...
 
+        # Correct:
+        class Test:
+            @staticmethod
+            def method(): ...
+
         # Wrong:
         class Test:
             def method(): ...
 
     .. versionadded:: 0.7.0
-    .. versionchanged:: 0.11.0
+    .. versionchanged:: 1.0.1
```

Comment:
```suggestion
    .. versionchanged:: 1.1.0
```

+ specify the change.